src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -2650,6 +2768,75 @@ namespace OutputReportPredefined {
+
+        pdrThermalResilience = newPreDefReport("ThermalResilienceSummary", "ThermR", "Annual Thermal Resilience Summary");
+
+        pdstHIHours = newPreDefSubTable(pdrThermalResilience, "Heat Index Hours");
+        pdchHIHourSafe = newPreDefColumn(pdstHIHours, "Safe (≤ 26.7°C) [Hours]");
+        pdchHIHourCaution = newPreDefColumn(pdstHIHours, "Caution (26.7, 32.2°C] [Hours]");
+        pdchHIHourExtremeCaution = newPreDefColumn(pdstHIHours, "Extreme Caution (32.2, 39.4°C] [Hours]");
+        pdchHIHourDanger = newPreDefColumn(pdstHIHours, "Danger (39.4, 51.7°C] [Hours]");
+        pdchHIHourExtremeDanger = newPreDefColumn(pdstHIHours, "Extreme Danger (> 51.7°C) [Hours]");
+
+        pdstHIOccuHours = newPreDefSubTable(pdrThermalResilience, "Heat Index OccupantHours");
+        pdchHIOccuHourSafe = newPreDefColumn(pdstHIOccuHours, "Safe (≤ 26.7°C) [OccupantHours]");
+        pdchHIOccuHourCaution = newPreDefColumn(pdstHIOccuHours, "Caution (26.7, 32.2°C] [OccupantHours]");
+        pdchHIOccuHourExtremeCaution = newPreDefColumn(pdstHIOccuHours, "Extreme Caution (32.2, 39.4°C] [OccupantHours]");
+        pdchHIOccuHourDanger = newPreDefColumn(pdstHIOccuHours, "Danger (39.4, 51.7°C] [OccupantHours]");
+        pdchHIOccuHourExtremeDanger = newPreDefColumn(pdstHIOccuHours, "Extreme Danger (> 51.7°C) [OccupantHours]");
+
+        pdstHumidexHours = newPreDefSubTable(pdrThermalResilience, "Humidex Hours");
+        pdchHumidexHourLittle = newPreDefColumn(pdstHumidexHours, "Little to no Discomfort (≤ 29) [Hours]");
+        pdchHumidexHourSome = newPreDefColumn(pdstHumidexHours, "Some Discomfort (29, 40] [Hours]");
+        pdchHumidexHourGreat = newPreDefColumn(pdstHumidexHours, "Great Discomfort; Avoid Exertion (40, 45] [Hours]");
+        pdchHumidexHourDanger = newPreDefColumn(pdstHumidexHours, "Dangerous (45, 50] [Hours]");
+        pdchHumidexHourStroke = newPreDefColumn(pdstHumidexHours, "Heat Stroke Quite Possible (> 50) [Hours]");
+
+        pdstHumidexOccuHours = newPreDefSubTable(pdrThermalResilience, "Humidex OccupantHours");
+        pdchHumidexOccuHourLittle = newPreDefColumn(pdstHumidexOccuHours, "Little to no Discomfort (≤ 29) [OccupantHours]");
+        pdchHumidexOccuHourSome = newPreDefColumn(pdstHumidexOccuHours, "Some Discomfort (29, 40] [OccupantHours]");
+        pdchHumidexOccuHourGreat = newPreDefColumn(pdstHumidexOccuHours, "Great Discomfort; Avoid Exertion (40, 45] [OccupantHours]");
+        pdchHumidexOccuHourDanger = newPreDefColumn(pdstHumidexOccuHours, "Dangerous (45, 50] [OccupantHours]");
+        pdchHumidexOccuHourStroke = newPreDefColumn(pdstHumidexOccuHours, "Heat Stroke Quite Possible (> 50) [OccupantHours]");
+
+        pdstHeatingSETHours = newPreDefSubTable(pdrThermalResilience, "Heating SET Hours");
+        pdchHeatingSETHours = newPreDefColumn(pdstHeatingSETHours, "SET ≤ 12.2°C Hours (°C)");
+        pdchHeatingSETOccuHours = newPreDefColumn(pdstHeatingSETHours, "SET ≤ 12.2°C OccupantHours (°C)");
```

Comment:
I assume these are also hours, please add [hr]. Actually in general, all the "[Hours]" should just be "[hr]"

Reply:
Okay. Thanks.